### PR TITLE
refac: V3 has its own models of Batch

### PR DESCRIPTION
**Pull request by lasrinnil**
## References

* #972

### CODE REVIEW

File: source/dotnet/wholesale-api/WebApi/V3/Batch/BatchDto.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    long? RunId,
+    Guid BatchId,
+    DateTimeOffset PeriodStart,
+    DateTimeOffset PeriodEnd,
+    string Resolution,
+    string Unit,
+    DateTimeOffset? ExecutionTimeStart,
+    DateTimeOffset? ExecutionTimeEnd,
+    BatchState ExecutionState,
+    bool AreSettlementReportsCreated,
```
Comment (BjarkeMeier) [inline source/dotnet/wholesale-api/WebApi/V3/Batch/BatchDto.cs:32]: Needed/used?

File: source/dotnet/wholesale-api/WebApi/V3/Batch/BatchDtoMapper.cs [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace Energinet.DataHub.Wholesale.WebApi.V3.Batch;
+
+public static class BatchDtoMapper
+{
+    public static BatchDto Map(Batches.Interfaces.Models.BatchDto batchDto)
```
Comment (BjarkeMeier) [inline source/dotnet/wholesale-api/WebApi/V3/Batch/BatchDtoMapper.cs:19]: Observation: There are no tests of the mappers you're adding.

**Review by BjarkeMeier — APPROVED** [review]

File: source/dotnet/wholesale-api/WebApi/V3/Batch/BatchDto.cs (hunk 1 above)
Comment (lasrinnil) [reply source/dotnet/wholesale-api/WebApi/V3/Batch/BatchDto.cs:32]: Yes - this way all models in the V3 API are separated from the models exposed by the modules.

File: source/dotnet/wholesale-api/WebApi/V3/Batch/BatchDtoMapper.cs (hunk 2 above)
Comment (lasrinnil) [reply source/dotnet/wholesale-api/WebApi/V3/Batch/BatchDtoMapper.cs:19]: made em'

File: source/dotnet/wholesale-api/WebApi.UnitTests/WebApi/V3/BatchDtoMapperTests.cs [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    [Theory]
+    [InlineAutoMoqData]
+    public static void MapDto_Returns_correct(Batches.Interfaces.Models.BatchDto source)
+    {
+        // Act
+        var actual = BatchDtoMapper.Map(source);
+
+        // Assert
+        actual.ProcessType.Should().Be(ProcessTypeMapper.Map(source.ProcessType));
+        actual.ExecutionState.Should().Be(BatchStateMapper.MapState(source.ExecutionState));
```
Comment (BjarkeMeier) [inline source/dotnet/wholesale-api/WebApi.UnitTests/WebApi/V3/BatchDtoMapperTests.cs:33]: Here you're testing actual against expected but they are both generated by the same code.
How about setting these two enums explicitly (using `.SetPrivateProperty(...)`)?

**Review by BjarkeMeier — APPROVED** [review]

**Pull request merged**